src/s3/util/fileViewerManager.ts

```diff
@@ -0,0 +1,44 @@
+/*!
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ext } from '../../shared/extensionGlobals'
+import { S3FileNode } from '../explorer/s3FileNode'
+import { Window } from '../../shared/vscode/window'
+import { showOutputMessage } from '../../shared/utilities/messages'
+import { OutputChannel } from 'vscode'
+export class FileViewerManager {
```

Comment:
naming: is this intended to be used for non-S3 purposes in the future? If not, `FileViewerManager` name is too generic. And more confusing when it is in a global context (`ext.fileViewerManager`). 
```suggestion
export class S3FileViewerManager {
```

Reply:
I don't think it is intended to be used for non-S3 purposes